docs/source/index.rst

```diff
@@ -63,7 +53,10 @@ such topics as:
 
    cpp/index
    python/index
-   java/index
+   `Java <https://arrow.apache.org/docs/java/>`_
```

Comment:
Why are you using this syntax? This makes docs incorrect.
https://framapic.org/gYWKmMs30com/kHYNqQJCHhf7.png